### PR TITLE
Xip-patch IP address fix

### PR DESCRIPTION
**Pull request by kfurgol**
**Description**

Changes proposed in this pull request:

- Xip-patch will now look for external ip address passed in env variable

**Related issue(s)**

### CODE REVIEW

File: components/xip-patch/README.md [hunk 1]
```diff
@@ -10,10 +10,11 @@ This component must be installed after `cluster-essentials` the component and be
 
 ## Usage
 
-You can't configure this component through environment variables.
+The patch accepts the **EXTERNAL_PUBLIC_IP** environment variable which is manually reserved external IP address. If this
```
Comment (klaudiagrz) [inline components/xip-patch/README.md]:
```suggestion
The patch accepts the **EXTERNAL_PUBLIC_IP** environment variable which is manually reserved for the external IP address. If this 
```

File: components/xip-patch/README.md [hunk 2]
```diff
@@ -10,10 +10,11 @@ This component must be installed after `cluster-essentials` the component and be
 
 ## Usage
 
-You can't configure this component through environment variables.
+The patch accepts the **EXTERNAL_PUBLIC_IP** environment variable which is manually reserved external IP address. If this 
+variable is not set, the script tries to get external IP address from the `istio-ingressgateway` service.
```
Comment (klaudiagrz) [inline components/xip-patch/README.md]:
```suggestion
variable is not set, the script tries to get the external IP address from the `istio-ingressgateway` service.
```

File: components/xip-patch/README.md (hunk 1 above)
Comment (sjanota) [reply components/xip-patch/README.md]: I would rather suggest:
```suggestion
The patch accepts the **EXTERNAL_PUBLIC_IP** environment variable which must hold manually reserved external IP address. If this 
```

Comment (klaudiagrz) [inline components/xip-patch/README.md]:
```suggestion
The patch accepts the **EXTERNAL_PUBLIC_IP** environment variable which is a manually reserved, external IP address. If this 
```

File: components/xip-patch/README.md [hunk 3]
```diff
@@ -10,10 +10,11 @@ This component must be installed after `cluster-essentials` the component and be
 
 ## Usage
 
-You can't configure this component through environment variables.
+The patch accepts the **EXTERNAL_PUBLIC_IP** environment variable which must hold a manually reserved, external IP address. If this
```
Comment (klaudiagrz) [inline components/xip-patch/README.md]:
```suggestion
The patch accepts the **EXTERNAL_PUBLIC_IP** environment variable which must contain a manually reserved, external IP address. If this 
```

**Pull request merged**